src/auth/authSlice.js

```diff
@@ -0,0 +1,32 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const authSlice = createSlice({
+  name: 'auth',
+  initialState: {
+    authToken: localStorage.getItem('FITNESS_TRACKER_APP_ADMIN_TOKEN') || null,
```

Comment:
Pls remove this, no need to store the token in slice. Keeping it in **LocalStorage** is enough.